doc/input-output-reference/src/overview/group-unitary-equipment.tex

```diff
@@ -726,7 +726,7 @@ \subsubsection{Inputs}\label{inputs-1-046}
 
 \paragraph{Field: No Load Supply Air Flow Rate Ratio}\label{field-no-load-supply-air-flow-rate-ratio}
 
-This field defines the no load operating air flow rate when the system fan is specified to operate continuously. The allowed fractions are between 0 and 1 with a default value of 1. The no load air flow rate will be calculated as this fraction multiplied by the minimum of the cooling and heating supply air flow rate. If the cooling or heating coil is not present, this fraction is multiplied by the operating supply air flow rate.
+This field defines the no load operating air flow rate when the system fan is specified to operate continuously. The allowed fractions are between 0 and 1 with a default value of 1. This fraction is set to the mimumum of heating and cooling operation lowest speed supply air flow fraction. The no load air flow rate will be calculated as this fraction multiplied by the minimum of the cooling and heating high speed supply air flow rate. If the cooling or heating coil is not present, this fraction is multiplied by the operating supply air flow rate.
```

Comment:
"This fraction is set to the minimum . . . " implies that it defaults or autosized to that. But if I understand correctly, the way things are now, the user has to remember to set this to the appropriate fraction, correct?
If I'm following correctly, then perhaps say "This fraction is usually set to the minimum . . . " ?

Reply:
If user has specified the no-load fraction, then user-specified value is used; otherwise, the idle flow fraction is defaulted to lowest speed fraction, which is the speed ratio for multi-speed DX coils. For variable speed DX coils, the no-load flow fraction is set to the lowest speed air flow ratio since we do not have *UnitarySystemPerformance:Multispeed* object.  I will fix the text as suggested.